lib/tasks/staging_seeds.rake

```diff
@@ -0,0 +1,63 @@
+# frozen_string_literal: true
+
+namespace :comfy do
+  
+  # Locally stored seeds - assumes you already have the local folder - 
+  # it won't create it 
+  LOCAL = (ComfortableMexicanSofa.config.seeds_path + '/protected-planet').freeze
+  REMOTE = 'ProtectedPlanet/current/db/cms_seeds/protected-planet'.freeze
+  PP_STAGING = 'new-web.pp-staging.linode.protectedplanet.net'.freeze
+  PP_PRODUCTION = 'new-web.pp-production.linode.protectedplanet.net'.freeze
+  PP_USER = 'wcmc'.freeze
+
+  desc "Import CMS Seed data from staging"
+  task :staging_import => :environment do |_t| 
+    
+    puts "Importing CMS Seed data from Staging Folder to #{LOCAL} ..."
+
+    puts question = "What would you like to import? 'All/Files/Layouts/Pages' or 'Nothing' to quit"
+    valid_answers = ['All', 'Files', 'Layouts', 'Pages', 'Nothing']
+    answer = STDIN.gets.chomp.downcase.capitalize
```

Comment:
I think it would be better to have one of those passed as an argument (downcased) to the task instead of asking the question at runtime as it would be quicker and checks would be easier. What do you think? @stanleypliu

Reply:
I personally have always liked the idea of a router interface as it feels more natural to me to input a command and then have the program execute what I just typed in. Plus, it also becomes more apparent to the user what can be passed to the task and what can't, as the task will inform them if their input is invalid.

Reply:
Ok, we can leave it like that. I was a bit concerned about the fact that people could run this task assuming it already does everything and leave, without noticing an input is required :) But it's fine like this as well

Reply:
You might have a point actually, if we want to run this on production, do we want to automate this so that it'll function without having to wait for user input?

Reply:
Potentially yeah.
You could even make it work both ways; for example you could make a check where, if arguments are provided, you won't ask for user input, while it does ask otherwise.

Reply:
That's the approach I went for ^